stylesheets/_component.select2.scss

```diff
@@ -255,37 +255,59 @@
 
 
 .select2-container--default .select2-selection--multiple .select2-selection__choice {
-    display: inline-block;
-    vertical-align: baseline;
-    box-sizing: content-box;
-    background-color: #dedede;
-    border-color: #dedede;
+    background-color: color(gray, lightest);
+    border-color: color(gray, lightest);
+    border-radius: $border-radius;
     border-width: 0;
-    color: #000;
-    border-radius: 4px;
+    box-sizing: content-box;
+    color: color(black);
+    display: inline-block;
     float: left;
     font-weight: 500;
     line-height: 34px;
     margin: 6px 0 0 5px;
-    padding: 0 0 0 10px;
-    text-align: center;
+    max-width: 86%;
```

Comment:
Super minor comment, something I raised on another PR, could this potentially be a use case for a Magic Number comment (https://sass-guidelin.es/#magic-numbers)? Certainly an interpretation thing, as it could be argued where the line is between an explicit value and a "magic number", for me if I came to work on this it'd be nice to know how we arrived at `86%` so as to know what the implications of changing this would be.